### PR TITLE
Update for Samsung Internet 11.2 and 12.0

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
Made using the mirroring script

### CODE REVIEW

**Comment by Elchi3** [comment]
Thanks for the PR @AdaRoseCannon! :tada: Can you comment on why we should have 11.2 as opposed to mark things as supported in 11.0? Isn't 11.2 just a minor version without new features?
This came up in https://github.com/mdn/browser-compat-data/pull/6099

**Comment by AdaRoseCannon** [comment]
In this case it is us implementing a feature early. It would otherwise be
out in 12.0.

File: api/WebGLRenderingContext.json [hunk 1]
```diff
@@ -5693,7 +5693,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": "12.0"
```
Comment (Elchi3) [inline api/WebGLRenderingContext.json:5696]: This is another XR API, is it 12.0 or 11.2?

Comment (AdaRoseCannon) [reply api/WebGLRenderingContext.json:5696]: Nice spot, thank you!!

File: api/Navigator.json [hunk 2]
```diff
@@ -1051,6 +1051,7 @@
             },
             "samsunginternet_android": {
               "version_added": true,
+              "version_removed": false,
```
[the remark this answers is not in the document]
Comment (AdaRoseCannon) [reply api/Navigator.json:1054]: It seems it was only available in the GearVR version of the browser, I think perhaps it's best to mark it `version_added: false`

**Pull request merged**